### ISSUE TITLE
Having problemas with Original combination and rank panel.

### ISSUE THREAD

**Issue by mabecabrera** [SpeciesFileGroup/taxonworks#3605]
### Steps to reproduce the bug

```bash
1. In edit taxon name task with the subspecies Tegra novaehollandiae novaehollandiae (Haan, 1843) https://sfg.taxonworks.org/tasks/nomenclature/new_taxon_name?taxon_name_id=911016.
2. When I edit the original combination, although I drag the subspecies name to the species rank to create the original name that has a species rank, it keeps on nameing a subspecies instead of a species.
```

### Screenshot

![image](https://github.com/SpeciesFileGroup/taxonworks/assets/63611746/f1186c23-2854-4992-97db-6575c6a90678)

### Environment

Production

### Version

 v0.35.1

### Browser Used

chrome

**Comment by proceps** [comment]
you used misspelling instead of the species name. It is fixed now:
![image](https://github.com/SpeciesFileGroup/taxonworks/assets/5597748/ac3bb55e-2023-4404-b5c1-22982348f57a)

**Comment by typophyllum** [comment]
The original combination is the misspelling.

**Comment by proceps** [comment]
It is correctly presented now "novaehollandiae" is a corrected original combination of the taxon. "novae hollandiae" is an incorrect original spelling. You have two protonyms. Each protonym should have relationship to self as a part of the original combination.

**Comment by mabecabrera** [comment]
Again. Trying to create the Petasia grisea name for original rank and combination but always getting a subspecies. The same thing that happened to me before and in the session had no trouble with.

<img width="1129" alt="image" src="https://github.com/SpeciesFileGroup/taxonworks/assets/63611746/59acc806-2d80-4502-99f2-6aaed49488cb">

This is what I am trying to recreate.
http://orthoptera.archive.speciesfile.org/Common/basic/Taxa.aspx?TaxonNameID=1120673

This is the url of TW.
https://sfg.taxonworks.org/tasks/nomenclature/new_taxon_name?taxon_name_id=935560

I have been into this point many times... is there any chance there is a bug or something? I could not show you this last time.

**Comment by proceps** [comment]
If the original combination is Petasia grisea, than the protonym name should be grisea, not griseus. Protonym should always should be correct original spelling of the original combination epithet. Also remember, the last element of the original combination should reference self. If not sure, you are on the protonym id 935560, you can just enter this number into the original species field. This should apply to both species and subspecie name. In fact you can just create correct original combination for subspecies, than go to species page, delete original combination and use soft validation to copy the original combination.

**Comment by mabecabrera** [comment]
Let me see if I follow you. 
I have the protonym Petasia grisea created. It is the number 935559. Then I have to create the subspecies. Although the subspecies epithet is "griseus" you are telling me to create it as "grisea". That is the original gender of the protonym. I think this is what I have been doing wrong (Or comes up wrong from the migration because of some incoherence in the old OSF data) and shows up with this situations when an original combination insists to have another name different from what i am completing in the corresponding fields. I just corrected the protonym and it got solved.

I will pay attention to that! Thank you so much for your help and patience.
